tests/CppUTestExt/IEEE754PluginTest.cpp

```diff
@@ -27,59 +27,67 @@
 
 /* The IEEE754ExceptionFlags plugin by definition requires C++11 */
 
-#ifdef CPPUTEST_COMPILER_FULLY_SUPPORTS_CXX11
-
 #include "CppUTest/CommandLineTestRunner.h"
 #include "CppUTest/TestHarness.h"
 #include "CppUTest/TestRegistry.h"
 #include "CppUTest/TestTestingFixture.h"
 #include "CppUTestExt/IEEE754ExceptionsPlugin.h"
-#include <cfenv>
 
 extern "C" { 
     #include "IEEE754PluginTest_c.h"
 }
 
+#if CPPUTEST_USE_STD_C_LIB
```

Comment:
I guess it's not such a big deal. My bigger problem is that the Watcom linker doesn't seem to find the functions :-(